core/sts-3.3.md

```diff
@@ -108,7 +108,19 @@ unable to do anything about except retrying at a later time.
 This is to ensure that it is not possible to fool users into allowing a
 man-in-the-middle attack.
 
-## Security
+## Security considerations
+
+### Acceptance of STS policy
+
+STS policies SHOULD ONLY be accepted for connections where the TLS certificate
+chain can be validated entirely, to either a user-specified root or the system
+root certificate bundle.
+
+### Visual feedback of STS upgraded connections
+
+Clients SHOULD NOT display visual feedback indicating that an STS-upgraded TLS
+connection has any enhanced security over a plaintext connection unless a user
+explicitly trusts the certificate.
```

Comment:
OK, so can the client provide visual feedback that the TLS connection has been verified by the trusted root? But then, if you're a user who doesn't know how to manually verify a certificate, isn't the fact that it's verified and implicitly trusted by your system a security enhancement?

Are we allowed to show a lock icon but just not make it green? How are we allowed to indicate a verified trusted root connection vs plain text. They *are* different.